src/ci/docker/x86_64-gnu-distcheck/Dockerfile

```diff
@@ -21,3 +21,10 @@ RUN sh /scripts/sccache.sh
 ENV RUST_CONFIGURE_ARGS --build=x86_64-unknown-linux-gnu --set rust.ignore-git=false
 ENV SCRIPT python2.7 ../x.py test distcheck
 ENV DIST_SRC 1
+
+# The purpose of this builder is to test that we can `./x.py test` successfully
+# from a tarball, not to test LLVM/rustc's own set of assertions. These cause a
+# significant hit to CI compile time (over a half hour as observed in #61185),
+# so disable assertions for this builder.
+ENV NO_LLVM_ASSERTIONS=1
+ENV NO_DEBUG_ASSERTIONS=1
```

Comment:
Wrong syntax here? Looking at the above we just want a space not `=`?

Reply:
It'll work either way, this is the syntax for multiple variables in one line wheras the previous is for one variable with a value that possibly contains spaces